lib/PreviewProvider.js

```diff
@@ -179,7 +179,7 @@ PreviewProvider.prototype = {
       .map(link => {
         const sanitizedURL = this._sanitizeURL(link.url);
         const cacheKey = this._createCacheKey(sanitizedURL);
-        return Object.assign({}, link, {sanitizedURL, cacheKey});
+        return Object.assign({}, link, {sanitized_url: sanitizedURL, cache_key: cacheKey});
```

Comment:
.map(link => {
  return Object.assign({}, link, {
    sanitized_url: this._sanitizeURL(link.url), 
    cache_key: this._createCacheKey(sanitizedURL)
  });
})